egs/swbd/s5c/local/chain/tuning/run_tdnn_blstm_1b.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+
+# tdnn_blstm_1b is same as tdnn_blstm_1b, but with the more tdnn layers
```

Comment:
At this point I'm preferring not to commit experiments that were not helpful... it's nice to have a record as a PR of what happened, but I'm concerned about permanently increasing the size of the repo.